lib/index.js

```diff
@@ -59,9 +59,12 @@ export async function applySession(
     set: store.set,
     get: store.get,
     unset: store.unset,
-    async save() {
+    async save(cookieOptionsOverrides = {}) {
```

Comment:
Let's use the same option name as init: `.save({cookieOptions: {...}})`, can you make this change?